Terminal.Gui/Input/Key.cs

```diff
@@ -972,17 +973,18 @@ out parsedInt
 
     #endregion
 
-    private static Rune _shortcutDelimiter = new ('+');
+    private static Rune _separator = new ('+');
 
     /// <summary>Sets or gets the shortcut delimiter separator. The default is "+".</summary>
-    public static Rune ShortcutDelimiter
+    [SerializableConfigurationProperty (Scope = typeof (SettingsScope))]
+    public static Rune Separator
     {
-        get => _shortcutDelimiter;
+        get => _separator;
         set
         {
-            if (_shortcutDelimiter != value)
+            if (_separator != value)
             {
-                _shortcutDelimiter = value == default (Rune) ? new ('+') : value;
+                _separator = value == default (Rune) ? new ('+') : value;
```

Comment:
I don't think this test for default is needed. _separator is set to "+" above.

Reply:
I added a unit test that proves it's needed to test for `Rune` default, otherwise `Ctrl+A` will only print `Ctrl`.